Documentation/Api/RestApi.apib

```diff
@@ -35,15 +34,16 @@ require authentication.
 
         {
             "id": 241,
-            "loginToken": "2cfe100561473c6cdd99c9e2f26fa974"
+            "key": "2cfe100561473c6cdd99c9e2f26fa974",
+            "expiry": "2017-07-20T18:22:48+00:00"
         }
 
 + Response 401 (application/json)
 
     + Body
 
         {
-            "code": 1,
+            "code": 1500567098798,
             "message": "Not authorized",
             "description": "The user name and password did not match any existing user."
```

Comment:
Anyway, the fields in the response JSON depend on the request, e.g. an error (which is recognizable by the HTTP status code) will always have an error code, a message and a description, while a successful request (e.g., code 201 for "created") will have different fields.